src/app/containers/Image/index.jsx

```diff
@@ -52,6 +53,7 @@ const ImageContainer = ({ blocks }) => {
   const ratio = (height / width) * 100;
   const rawImageSrc = getRawImageSrc(originCode, locator);
   const srcSet = createSrcset(originCode, locator, width);
+  const lazyLoad = index > LAZYLOAD_FROM_BLOCK;
```

Comment:
What does this evaluate to when `index` isn't defined? Should `index` have a default value?

Reply:
Have set a default to 0.